awx/main/analytics/collectors.py

```diff
@@ -277,7 +277,7 @@ def _copy_table(table, query, path):
                                  JOIN main_job ON main_unifiedjob.id = main_job.unifiedjob_ptr_id
                                  JOIN django_content_type ON main_unifiedjob.polymorphic_ctype_id = django_content_type.id
                                  JOIN main_project ON main_project.unifiedjobtemplate_ptr_id = main_job.project_id
```

Comment:
Do we still need to include `main_project` in the `JOIN` now?

Reply:
Oh! maybe not. I also don't see any reference to `main_job` other than the JOIN specifically for this.

@rooftopcellist do you think it would be safe to remove both of these lines?

```sql
JOIN main_job ON main_unifiedjob.id = main_job.unifiedjob_ptr_id
...
JOIN main_project ON main_project.unifiedjobtemplate_ptr_id = main_job.project_id
```

I suspected that I might be missing something after editing this code.